<?xml version="1.0" encoding="UTF-8"?>
<interface xmlns="http://xml.phpdox.net/src" full="Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface" namespace="Symfony\Contracts\HttpClient\Exception" name="TransportExceptionInterface" abstract="false" final="false" start="19" end="21">
  <file path="//var/www/html/vendor/symfony/http-client-contracts/Exception" file="TransportExceptionInterface.php" realpath="/var/www/html/vendor/symfony/http-client-contracts/Exception/TransportExceptionInterface.php" size="461" time="2022-03-13T20:07:29+00:00" unixtime="1647202049" sha1="4e7d0c2c45ed79d8d63b5d603dfc44042ce36e3a" relative="ar/www/html/vendor/symfony/http-client-contracts/Exception/TransportExceptionInterface.php"/>
  <docblock>
    <description compact="When any error happens at the transport level."/>
    <author value="Nicolas Grekas &lt;p@tchwork.com&gt;"/>
  </docblock>
  <extends full="Symfony\Contracts\HttpClient\Exception\ExceptionInterface" namespace="Symfony\Contracts\HttpClient\Exception" name="ExceptionInterface"/>
  <parent full="Symfony\Contracts\HttpClient\Exception\ExceptionInterface" namespace="Symfony\Contracts\HttpClient\Exception" name="ExceptionInterface">
    <extends full="Throwable" namespace="" name="Throwable"/>
  </parent>
  <parent full="Throwable" namespace="" name="Throwable">
    <method name="getMessage" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Gets the message"/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="getCode" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Gets the exception code"/>
      <return type="int"/>
    </docblock>
  </method>
    <method name="getFile" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Gets the file in which the exception was created"/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="getLine" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Gets the line on which the object was instantiated"/>
      <return type="int"/>
    </docblock>
  </method>
    <method name="getTrace" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Gets the stack trace"/>
      <return type="array"/>
    </docblock>
  </method>
    <method name="getTraceAsString" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Gets the stack trace as a string"/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="getPrevious" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Returns the previous Throwable"/>
      <return type="Throwable"/>
    </docblock>
  </method>
    <method name="__toString" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Gets a string representation of the thrown object"/>
      <return type="string"/>
    </docblock>
  </method>
  </parent>
  <extenders>
    <extender full="Symfony\Contracts\HttpClient\Exception\TimeoutExceptionInterface" namespace="Symfony\Contracts\HttpClient\Exception" name="TimeoutExceptionInterface"/>
  </extenders>
  <implementors>
    <class full="Symfony\Component\HttpClient\Exception\TimeoutException" namespace="Symfony\Component\HttpClient\Exception" name="TimeoutException"/>
    <class full="Symfony\Component\HttpClient\Exception\InvalidArgumentException" namespace="Symfony\Component\HttpClient\Exception" name="InvalidArgumentException"/>
    <class full="Symfony\Component\HttpClient\Exception\TransportException" namespace="Symfony\Component\HttpClient\Exception" name="TransportException"/>
  </implementors>
</interface>
